### ISSUE TITLE
Accept [filter_condition] in AbilitiesWML tags

### ISSUE THREAD

**Issue by ProditorMagnus** [wesnoth/wesnoth#2766]
Currently conditions need to be done though StandardUnitFilter formula or lua_function.

Use case: https://github.com/ProditorMagnus/Ageless-for-1-11/issues/13

**Comment by CelticMinstrel** [comment]
Good idea, though is there any situation where an ability filter might need to be tested without access to the game state? (I can't think of any, though. And it's a moot point if the Lua kernel is also part of the game state.)

**Comment by jostephd** [comment]
Does lua_function work in AbilitiesWML? There's no mention of it on https://wiki.wesnoth.org/AbilitiesWML#Common_keys_and_tags_for_every_ability

**Comment by ProditorMagnus** [comment]
Even if it does, that is not a solution for this issue.
